clang/include/clang/Basic/Attr.td

```diff
@@ -1716,6 +1716,7 @@ def IntelFPGABankWidth : Attr {
   let Subjects = SubjectList<[IntelFPGAConstVar, IntelFPGALocalStaticSlaveMemVar,
                               Field], ErrorDiag>;
   let LangOpts = [SYCLIsDevice, SYCLIsHost];
+  let HasCustomTypeTransform = 1;
```

Comment:
I don't think that is needed.  `HasCustomTypeTransform` flag affects only statement attributes, while all the attribute below are declaration attributes.